documentation/src/main/asciidoc/ch07.asciidoc

```diff
@@ -1,3 +1,324 @@
 [[chapter-valueextraction]]
 == Value extraction
 
+Value extraction is the process extracting values from a container so that
+they can be validated.
+
+It is used when dealing with <<container-element-constraints,container element
+constraints>> and <<example-cascaded-validation-list,cascaded validation inside
+containers>>.
+
+[[section-valueextraction-builtinvalueextractors]]
+=== Built-in value extractors
+
+Hibernate Validator comes with built-in value extractors for the usual Java container
+types so, apart if you are using your own custom container types (or the ones
```

Comment:
apart => except? Not sure about that, but I've never heard "apart if ...", always "apart from -ing".